src/main/java/com/google/step/finscholar/data/ServletConstantValues.java

```diff
@@ -20,11 +20,23 @@
     public static final String INDEX_PATH = "/index.html";	
     // The default value for undefined fields.	
     public static final String DEFAULT_VALUE = "";
-    
-    // Constants for exceptions when retrieving data from the Firestore database.
+
+    // Constants for the different Firestore Collection names.
+    public static final String SCHOLARSHIP_COLLECTION_NAME = "scholarships";
+    public static final String COLLEGE_COLLECTION_NAME = "colleges";
+    public static final String FORUM_POST_COLLECTION_NAME = "forumPosts";
+    public static final String FORUM_COMMENT_COLLECTION_NAME = "forumCommments";
+
+    // Constants for exceptions when retrieving data from the Firestore Database.
+    public static final String UNABLE_TO_WRITE_TO_FIRESTORE = "Unable to write new object to collection with name: ";
+    public static final String UNABLE_TO_READ_FROM_FIRESTORE = "Unable to retrieve data from Firestore.";
+    public static final String DOCUMENT = "Document with ID: ";
+    public static final String NEW_DOCUMENT_ADDED = "New Document added to: ";
+    public static final String AT = " at: ";
+    public static final String DNE = " does not exist.";
     public static final String UNABLE_TO_LOAD_FIREBASE = "Unable to load firebase: ";
```

Comment:
Happy to see a file full of constants! In a future PR, you should break up this file into several. Maybe one for ErrorMessages, one for Paths, and one for FireStoreFields.

Reply:
Will do!